src/tests/tests/utils.js

```diff
@@ -477,6 +477,11 @@ QUnit.test("findCommonSubstrings", assert => {
     "returned matches have length greater than or equal to 8"
   );
 
+  assert.notEqual(
+    utils.findCommonSubstrings('www.foo.bar', 'www.foob.ar')[0] === 'www.foo',
```

Comment:
I think you got the parameters to `assert.notEqual` mixed up here.

Reply:
It would be more to the point to verify here that there are no results, not that the first result isn't equal to some hypothetical result.